vm/transaction.go

```diff
@@ -58,3 +50,171 @@ func marshalTxn(txn core.Transaction) (json.RawMessage, error) {
 	}
 	return result, nil
 }
+
+type Transaction struct {
+	Hash                  *felt.Felt                   `json:"transaction_hash,omitempty" copier:"must,nopanic"`
```

Comment:
Do we still need the copier tag? We can remove it from the transaction struct in `starknet/transaction.go` I think.

Reply:
Removed.

Reply:
Actually, we can get rid of hash field all together.